app/src/main/java/school/cactus/cactusnotesmvvmsample/SignupViewModel.kt

```diff
@@ -1,9 +1,14 @@
 package school.cactus.cactusnotesmvvmsample
 
+import android.R
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 
-class SignupViewModel : ViewModel() {
+
+class SignupViewModel(application: Application) : AndroidViewModel(application) {
+    private val context = getApplication<Application>().applicationContext
```

Comment:
I think you don't need to use `getApplication<Application>()`, because you already have it: `application`